content/docs/reactivesearch/vue/list/MultiDropdownList.md

```diff
@@ -80,49 +85,107 @@ Example uses:
     show count of number of occurences besides an item. Defaults to `true`.
 -   **showSearch** `Boolean` [optional]
     whether to show a searchbox to filter the list items locally. Defaults to false.
--   **renderItem** `Function|slot-scope` [optional]
-    customize the rendered list via a function or slot-scope which receives the item label, count and isChecked & expects a JSX or String back. For example:
-
-```js
-    renderItem={({ label, count }) => (
-        <div>
-            {label}
-            <span style={{ marginLeft: 5, color: 'dodgerblue' }}>
-                {count}
-            </span>
-        </div>
-    )}
+-   **render** `Function|slot-scope` [optional]
+    an alternative callback function to `renderItem`, where user can define how to render the view based on all the data changes.
+    <br/>
+    It accepts an object with these properties:
+    -   **`loading`**: `boolean`
+        indicates that the query is still in progress
+    -   **`error`**: `object`
+        An object containing the error info
+    -   **`data`**: `array`
+        An array of results obtained from the applied query.
+    -   **`value`**: `array`
+        current selected values.
+    -   **`handleChange`**: `function`
+        A callback function can be used to mark the list value as selected.
+    -   **`downshiftProps`**: `object`
+        provides the following control props from `downshift` which can be used to bind list items with click/mouse events.
+        -   **isOpen** `boolean`
+            Whether the menu should be considered open or closed. Some aspects of the downshift component respond differently based on this value (for example, if isOpen is true when the user hits "Enter" on the input field, then the item at the highlightedIndex item is selected).
+        -   **getItemProps** `function`
+            Returns the props you should apply to any menu item elements you render.
+        -   **getItemEvents** `function`
+            Returns the events you should apply to any menu item elements you render.
+        -   **highlightedIndex** `number`
+            The index that should be highlighted.
+
+You can use render as a slot as shown below:
+
+<!-- prettier-ignore -->
+```html
+<MultiDropdownList
```

Comment:
@ShahAnuj2610 Can we use component names ad kebab case which is commonly used in Vue, please update it across all the components.

Reply:
done